lms/djangoapps/certificates/badge_handler.py

```diff
@@ -0,0 +1,134 @@
+"""
+BadgeHandler object-- used to award Badges to users who have completed courses.
+"""
+import hashlib
+import logging
+import requests
+
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.core.urlresolvers import reverse
+from lazy.lazy import lazy
```

Comment:
Even lazier, I just type lazy somewhere in the code, right-click, tell IntelliJ to import, and this is what it comes up with :P Good to know, though.